src/elements/content-preview/ContentPreview.js

```diff
@@ -550,6 +560,19 @@ class ContentPreview extends PureComponent<Props, State> {
         logger.onPreviewMetric(metrics);
     };
 
+    /**
+     * Event handler for viewer events from Preview. Currently only handles thumbnails open and close
+     * @param {Object} viewerEvent - the viewer event object
+     * @return {void}
+     */
+    onPreviewViewerEvent = ({ event: eventName }: PreviewViewerEvent): void => {
+        if (eventName === 'thumbnailsOpen') {
```

Comment:
Probably worth converting this to a switch to make future additions easier if you want to continue binding to 'viewerevent'. Also probably worth a constants list in this file of preview events we're using.

Reply:
will get rid of this per your suggestion